tests/modules/somod/somod.c

```diff
@@ -0,0 +1,85 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+#include <src/jv.h>
+#include <src/jq.h>
+#ifndef JQ_PLUGIN
+#error "JQ_PLUGIN not defined!"
+#endif
+#include <src/jq_plugin.h>
+
+static jv f_add1(jq_state *jq, jv input) {
+  if (jv_get_kind(input) == JV_KIND_NUMBER) {
```

Comment:
Here and below, shouldn't this plugin be calling the jv_* functions via the jq_state pointer?

Reply:
There's macros that do that!